autoyast/provision_sles.erb

```diff
@@ -124,6 +124,7 @@ cp /etc/resolv.conf /mnt/etc
 <% if salt_enabled %>
 <%= snippet 'saltstack_setup' %>
 <% end -%>
+
 <% if spacewalk_enabled -%>
```

Comment:
provision_rhel has this higher so it's able to install the packages if using Spacewalk etc.